CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+0.20.1 (2018-08-28)
```

Comment:
Please revert all the release related code, that is handled when I make a release.

Reply:
Ooops, my bad. Will do.